idega/fake-backend/__tests__/queries.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { wine } from './helpers/mocks/wine';
+import { get, getById, add, Wine } from '../queries';
+
+describe('get', () => {
+  it('returns a colection of wines', async () => {
+    const wines = await get();
+
+    expect(wines).toEqual(
+      expect.objectContaining<Wine>({} as Wine)
+    )
+  });
+})
+
+describe('getById', () => {
+  it('returns a wine by it\'s id', async () => {
+    const wines = await getById(1);
+
+    expect(wines).toEqual(
+      expect.objectContaining<Wine>({} as Wine)
+    )
+    expect(wines?.id).toBe(1)
```

Comment:
We should use strict assertion here:

```suggestion
    expect(wines!.id).toBe(1)
```